airbyte-server/src/main/java/io/airbyte/server/handlers/WorkspacesHandler.java

```diff
@@ -263,6 +268,18 @@ private static WorkspaceRead buildWorkspaceRead(final StandardWorkspace workspac
         .securityUpdates(workspace.getSecurityUpdates())
         .notifications(NotificationConverter.toApiList(workspace.getNotifications()))
         .defaultGeography(Enums.convertTo(workspace.getDefaultGeography(), Geography.class));
+    // Add read-only webhook configs.
+    try {
+      final WebhookOperationConfigs persistedConfigs = new ObjectMapper().treeToValue(
```

Comment:
wondering whether we can use some of the `Jsons` [methods](https://github.com/airbytehq/airbyte/blob/master/airbyte-commons/src/main/java/io/airbyte/commons/json/Jsons.java#L108) in order to simplify the exception handling